pkg/apis/lighthouse/v2alpha1/serviceimport_types.go

```diff
@@ -0,0 +1,116 @@
+package v2alpha1
```

Comment:
We cannot add the lighthouse APIs in our tree.

That lives on lighthouse.

I understand it's just to circunvent the limitation of the operator-sdk, and I understand if that's a step that we do during generation, I believe we shouldn't add them on the git tree.

@skitt  ^ thoughts?

Reply:
Yup, I’m hoping we can use the vendored dependencies to drive the generation somehow... Or Kubebuilder once we’ve upgraded to Operator SDK v1.

Reply:
@mangelajo @skitt should I close this PR for now? I will start working on #678 and then we can have this discussion again

Reply:
Yes, I suspect that’s the most sensible option — migrating to the Operator SDK involves re-creating the project anyway, and will resultin a flattened structure. Note too that I’m migrating most CRDs to the operator itself in https://github.com/submariner-io/submariner-operator/pull/685 which will solve a few issues for you ;-).

Reply:
sounds good